locales/en-US/app.ftl

```diff
@@ -95,7 +95,11 @@ CallNodeContextMenu--transform-collapse-resource =
 CallNodeContextMenu--transform-collapse-direct-recursion = Collapse direct recursion
     .title =
         Collapsing direct recursion removes calls that repeatedly recurse into
-        the same function.
+        the same function with no intermediate functions on the stack.
+CallNodeContextMenu--transform-collapse-indirect-recursion = Collapse indirect recursion
+    .title =
+        Collapsing indirect recursion removes calls that repeatedly recurse into
```

Comment:
```suggestion
        Collapsing indirect recursions removes calls that repeatedly recurse into
```

Reply:
The previously existing direct recursion description uses singular here though.

Reply:
Uh, I missed that. When you use this command, can you collapse more than one recursion or is it always one?

Reply:
I think the current wording throughout speaks of recursion as an a concept, which isn't countable. I guess it's possible to also count recursions, although it sounds odd (at least to me) and somewhat ambiguous: does one recursion refer to a single recursive function call or the stack of calls. It would almost always be multiple then.

A quick look on Google reveals there are profilers out there that call it "collapse recursion" and other that call it "collapse recursions". Making this change consistently requires a lot more changes that kind of go beyond the scope of this PR.

Reply:
That's a good point. Feel free to ignore the suggestion.